lib/metadata/MIQExtract/MIQExtract.rb

```diff
@@ -126,7 +126,7 @@ def getProfileData(&blk)
 
   def extract(category, &blk)
     xml = nil
-    Array.wrap(category).each do |c|
+    Array.wrap(category.presence).each do |c|
```

Comment:
if category were nil this would have been fine previously.  In the PR title did you mean "to handle blank case"?

Reply:
thanks